engine/installation/linux/docker-ee/centos.md

```diff
@@ -154,25 +158,30 @@ EE from the repository.
     ```bash
     $ yum list docker-ee.x86_64  --showduplicates | sort -r
 
-    docker-ee.x86_64  {{ minor-version }}.0.el7                               docker-ee-stable  
+    docker-ee.x86_64        {{ minor-version }}.ee.2-1.el7.centos         docker-ee-stable-17.06
```

Comment:
This part isn't necessarily correct, the `.ee.2` actually reflects the current `ee` version.

Every patched version of Docker ee will increment that number. I know that's a bit hard to document but the best thing that most businesses can do is just lock to a major version like 17.06 and then install the highest patch version of that major version.

You can find the latest patch version with the following command:

```
yum list docker-ee --show-duplicates | grep "17.06" | head | awk '{print $2}'
```

That way you're on the latest patch set with all the latest fixes as well.

Also the version names directly correlate with the minor-version they just have a substitution of `-` -> `.`

Example:
`17.06.1-ee-2` -> `17.06.1.ee.2`